src/libponyc/codegen/host.cc

```diff
@@ -206,6 +206,18 @@ void LLVMSetMetadataStr(LLVMValueRef inst, const char* str, LLVMValueRef node)
   unwrap<Instruction>(inst)->setMetadata(str, n);
 }
 
+void LLVMMDNodeReplaceOperand(LLVMValueRef parent, unsigned i, 
+  LLVMValueRef node)
+{
+  pony_assert(parent != NULL);
+  pony_assert(node != NULL);
+  pony_assert((int)i >= 0);
```

Comment:
This check isn't needed. `llvm::MDNode::replaceOperandWith` has an unsigned parameter so `i` will never be used as signed.